mmv1/third_party/terraform/tests/resource_compute_instance_group_manager_test.go.erb

```diff
@@ -368,23 +368,24 @@ func TestAccInstanceGroupManager_stateful(t *testing.T) {
 	target := fmt.Sprintf("tf-test-igm-%s", randString(t, 10))
 	igm := fmt.Sprintf("tf-test-igm-%s", randString(t, 10))
 	hck := fmt.Sprintf("tf-test-igm-%s", randString(t, 10))
+  network := fmt.Sprintf("tf-test-igm-%s", randString(t, 10))
```

Comment:
```suggestion
	network := fmt.Sprintf("tf-test-igm-%s", randString(t, 10))
```

Reply:
Thanks for pointing them out, locally I really don't see the indentation difference. Will try to correct them properly.